PlanAndRide/PlanAndRide.BusinessLogic/Review.cs

```diff
@@ -4,5 +4,54 @@ public class Review
     {
         public int Score { get; set; }
         public string Description { get; set; }
+
+        private string newScore;
+
+        public void NewReview()
+        {
+            //ocena
+
+            Console.WriteLine("Stórzmy recenzję!\n");
+            Console.WriteLine("Skala: 1-5;");
+            Console.WriteLine("1 - bardzo słaba, 5 - rewelacja");
+            newScore = Console.ReadLine().Trim();
+
+            while (newScore == "" || !int.TryParse(newScore, out int Score) || Score > 5 || Score <= 0)
+            {
+                Console.Write("Wprowadź ocenę 1-5:");
+                newScore = Console.ReadLine();
+                Console.WriteLine("");
+            }
+
+            Score = int.Parse(newScore);
+
+            Console.WriteLine($"\nTwoja ocenta to: {Score}\n");
+
+            //opis
+
+            Console.WriteLine("Czy chcesz dodać opis recenzji (wprowadź 't' lub 'n': ");
```

Comment:
tutaj prosisz o podanie t lub n, to czemu dalej sprawdzasz tez "tak" i "nie"?

Reply:
poozostawiłem tylko 't' i 'n'

Reply:
może zmienić to wszystko na:
"Wybierz:
1. tak
2. nie"
odczytać czy użytkownik wprowadził 1 lub 2 i wtedy np. switchem?

Reply:
poprawione - nowa metoda